### PR TITLE
Githubactions

### PR DESCRIPTION
**Pull request by balmas**
switches the qa build from travis to githubactions

Will add production and ftqa in a subsequent PR

### CODE REVIEW

File: github-build.mjs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  const buildInfo = generateBuildInfo(buildDT)
+  console.log(`Starting build ${buildInfo.name}`)
+
+  const baseVersion = pkg.version.split('-')[0]
+  const version = `${baseVersion}-${buildInfo.name}`
+  console.log(`Setting a package version to ${version}`)
+  let output
+  try {
+    if (buildInfo.branch === 'qa' || buildInfo.branch === 'production') {
+      console.info(`Installing alpheios-core`)
```
Comment (kirlat) [inline github-build.mjs:18]: Can we see console messages in GitHub Actions? Are those messages ever needed for GitHub?

File: github-build.mjs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    }
+  } catch (error) {
+    console.error('Components install failed:', error)
+    process.exit(1)
+  }
+  try {
+    const output = execSync(`npm version ${version} --no-git-tag-version --force`, { encoding: 'utf8' })
+  } catch (e) {
+    console.error('Cannot execute npm version:', e)
+    process.exit(1)
```
Comment (kirlat) [inline github-build.mjs:29]: Here is the same return code as before. Should we use a unique one?

Comment (balmas) [reply github-build.mjs:29]: Yes, this is the way it was in tagged-commit but I'll go ahead and fix it.

File: github-build.mjs (hunk 1 above)
Comment (balmas) [reply github-build.mjs:18]: We can see them. Although these console messages aren't needed for GitHub, I think they are helpful to keep for debugging.

**Pull request merged**